auctions/english-auction/project/Forc.toml

```diff
@@ -1,6 +1,2 @@
 [workspace]
-members = [
-  "./contracts/auction-contract",
-  "./contracts/auction-contract/tests/artifacts/asset",
-  "./contracts/auction-contract/tests/artifacts/NFT",
-]
+members = ["./contracts/auction-contract", "./contracts/test-artifacts//NFT"]
```

Comment:
```suggestion
members = ["./contracts/auction-contract", "./contracts/test-artifacts/NFT"]
```

Reply:
Will resolve in follow up PR, as this is a protected branch